src/tests/slot-composer-test.ts

```diff
@@ -132,15 +127,14 @@ recipe
     const slotComposer = arc.pec.slotComposer;
     const observer = new SlotTestObserver();
     slotComposer.observeSlots(observer);
+
     observer
         .newExpectations()
         .expectRenderSlot('List', 'root')
         .expectRenderSlot('List', 'root')
         .expectRenderSlot('ShowProduct', 'item')
         ;
-
-    await suggestion.instantiate(arc);
-    await arc.idle;
+    await arc.instantiate(suggestion.plan);
```

Comment:
just curious - why did you remove the `await arc.idle`?

Reply:
Because observer is going to wait until expectations are met (or until timeout), waiting for the arc to go idle isn't semantically different.